src/Resources/public/app.js

```diff
@@ -1 +1 @@
gth;)!1===r[a].apply(n[0],n[1])&&t.stopOnFalse&&(a=r.length,n=!1);t.memory||(n=!1),e=!1,o&&(r=n?[]:"")},c={add:function(){return r&&(n&&!e&&(a=r.length-1,s.push(n)),function e(n){g.each(n,(function(n,i){g.isFunction(i)?t.unique&&c.has(i)||r.push(i):i&&i.length&&"string"!==g.type(i)&&e(i)}))}(arguments),n&&!e&&l()),this},remove:function(){return g.each(arguments,(function(t,e){for(var n;(n=g.inArray(e,r,n))>-1;)r.splice(n,1),n<=a&&a--})),this},has:function(t){return t?g.inArray(t,r)>-1:r.length>0},empty:function(){return r&&(r=[]),this},disable:function(){return o=s=[],r=n="",this},disabled:function(){return!r},lock:function(){return o=s=[],n||(r=n=""),this},locked:function(){return!!o},fireWith:function(t,n){return o||(n=[t,(n=n||[]).slice?n.slice():n],s.push(n),e||l()),this},fire:function(){return c.fireWith(this,arguments),this},fired:function(){return!!i}};return c},g.extend({Deferred:function(t){var e=[["resolve","done",g.Callbacks("once memory"),"resolved"],["reject","fail",g.Callbacks("once memory"),"rejected"],["notify","progress",g.Callbacks("memory")]],n="pending",i={state:function(){return n},always:function(){return o.done(arguments).fail(arguments),this},then:function(){var t=arguments;return g.Deferred((function(n){g.each(e,(function(e,r){var s=g.isFunction(t[e])&&t[e];o[r[1]]((function(){var t=s&&s.apply(this,arguments);t&&g.isFunction(t.promise)?t.promise().progress(n.notify).done(n.resolve).fail(n.reject):n[r[0]+"With"](this===i?n.promise():this,s?[t]:arguments)}))})),t=null})).promise()},promise:function(t){return null!=t?g.extend(t,i):i}},o={};return i.pipe=i.then,g.each(e,(function(t,r){var s=r[2],a=r[3];i[r[1]]=s.add,a&&s.add((function(){n=a}),e[1^t][2].disable,e[2][2].lock),o[r[0]]=function(){return o[r[0]+"With"](this===o?i:this,arguments),this},o[r[0]+"With"]=s.fireWith})),i.promise(o),t&&t.call(o,o),o},when:function(t){var e,n,i,o=0,r=a.call(arguments),s=r.length,l=1!==s||t&&g.isFunction(t.promise)?s:0,c=1===l?t:g.Deferred(),u=function(t,n,i){return function(o){n[t]=this,i[t]=arguments.length>1?a.call(arguments):o,i===e?c.notifyWith(n,i):--l||c.resolveWith(n,i)}};if(s>1)for(e=new Array(s),n=new Array(s),i=new Array(s);o<s;o++)r[o]&&g.isFunction(r[o].promise)?r[o].promise().progress(u(o,n,e)).done(u(o,i,r)).fail(c.reject):--l;return l||c.resolveWith(i,r),c.promise()}}),g.fn.ready=function(t){return g.ready.promise().done(t),this},g.extend({isReady:!1,readyWait:1,holdReady:function(t){t?g.readyWait++:g.ready(!0)},ready:function(t){(!0===t?--g.readyWait:g.isReady)||(g.isReady=!0,!0!==t&&--g.readyWait>0||(j.resolveWith(s,[g]),g.fn.triggerHandler&&(g(s).triggerHandler("ready"),g(s).off("ready"))))}}),g.ready.promise=function(t){return j||(j=g.Deferred(),"complete"===s.readyState||"loading"!==s.readyState&&!s.documentElement.doScroll?i.setTimeout(g.ready):(s.addEventListener("DOMContentLoaded",L),i.addEventListener("load",L))),j.promise(t)},g.ready.promise();var M=function(t,e,n,i,o,r,s){var a=0,l=t.length,c=null==n;if("object"=
```

Comment:
?

Reply:
Changing the jQuery version modifies the compiled file, what do you see strange here?

Reply:
The diff looks weird, thought it was a mistake, all good then